requirements.txt

```diff
@@ -1 +0,0 @@
--e .
```

Comment:
Does this affect installation at all? and if that's the only line, should we just delete the file?

Reply:
I remember tom telling me this was necessary to keep, but I can't remember why

Reply:
I removed it and it installed just fine for me, I've never seen anything like this before. @TRManderson ?

Reply:
The answer is "it depends".

@gricey432, you're very used to doing `pip install -e .` from work stuff, but that's not the normal workflow for most python devs.

people using `pipenv` need the `requirements.txt`
people using `requirements.txt` need it
people using `setup.py` are covered
people using `pip install .` are covered

Reply:
But those first two methods won't work with tests_requires right? So having it there is a bit confusing.

Reply:
The `README.md` has been changed to tell people to use `pip`, so I'd say we don't need this anymore. Keeping it as simple as possible for a first-year to look around the repo and figure out what stuff is for